dtfs-central-api/api-tests/v1/mapping/mapDataModel.api-test.js

```diff
@@ -27,12 +27,6 @@ describe('Mapping cross data model fields', () => {
       expect(actual).toEqual(expected);
     });
 
-    it('Should return Bank name', () => {
```

Comment:
is there a test for the changes made here? or not needed in this function?

Reply:
It's been removed in the file it's testing so no longer required 👍  https://github.com/UK-Export-Finance/dtfs2/pull/556/files#diff-7cd24e836e4f30ffe948421527d808406f86585bcaeac4f4f46202c7049b8b09L13